src/core/types/State.ts

```diff
@@ -97,6 +97,10 @@ export interface VaultTransaction {
   expectedOutcome: TransactionOutcome | undefined;
 }
 
+export type IdToCreditPositionMap = {
```

Comment:
@kibagateaux would appreciate your opinion on taking this approach. By defining a type, I'm able to import that into different files where the mapping is utilized.